cps/kni.c

```diff
@@ -33,6 +33,9 @@
 #include "gatekeeper_lls.h"
 #include "gatekeeper_main.h"
 #include "kni.h"
+#include "elf.h"
```

Comment:
Let's add header `elf.h` immediately before header `kni.h` since this is the file `kni.c`.